tests/test_assets/genes_biodomains/input/biodomains_test_input.csv

```diff
@@ -1,37 +1,37 @@
-Autophagy,Au,Autophagy [Au],#9931fd,GO:0005776,autophagosome,ENSG00000161011
-Autophagy,Au,Autophagy [Au],#9931fd,GO:0000407,phagophore assembly site,ENSG00000161011
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0005743,mitochondrial inner membrane,ENSG00000000938
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0005758,mitochondrial intermembrane space,ENSG00000000938
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0098780,response to mitochondrial depolarisation,ENSG00000161011
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0005739,mitochondrion,ENSG00000161011
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0005739,mitochondrion,ENSG00000000938
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0000422,autophagy of mitochondrion,ENSG00000161011
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0010821,regulation of mitochondrion organization,ENSG00000161011
-Mitochondrial Metabolism,MM,Mitochondrial Metabolism [MM],#97cb98,GO:0000423,mitophagy,ENSG00000161011
-Synapse,Sy,Synapse [Sy],#329a33,GO:0045202,synapse,ENSG00000188157
-Synapse,Sy,Synapse [Sy],#329a33,GO:0050808,synapse organization,ENSG00000188157
-Synapse,Sy,Synapse [Sy],#329a33,GO:0045887,positive regulation of synaptic growth at neuromuscular junction,ENSG00000188157
-Synapse,Sy,Synapse [Sy],#329a33,GO:1900273,positive regulation of long-term synaptic potentiation,ENSG00000161011
-Synapse,Sy,Synapse [Sy],#329a33,GO:0005102,signaling receptor binding,ENSG00000000938
-Synapse,Sy,Synapse [Sy],#329a33,GO:0007213,G protein-coupled acetylcholine receptor signaling pathway,ENSG00000188157
-Synapse,Sy,Synapse [Sy],#329a33,GO:0007528,neuromuscular junction development,ENSG00000188157
-Synapse,Sy,Synapse [Sy],#329a33,GO:0045887,positive regulation of synaptic assembly at neuromuscular junction,ENSG00000188157
-Synapse,Sy,Synapse [Sy],#329a33,GO:0035255,ionotropic glutamate receptor binding,ENSG00000161011
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0005783,endoplasmic reticulum,ENSG00000161011
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0005796,Golgi lumen,ENSG00000188157
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0042254,ribosome biogenesis,ENSG00000290146
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0070475,rRNA base methylation,ENSG00000290146
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0070037,rRNA (pseudouridine) methyltransferase activity,ENSG00000290146
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0031625,ubiquitin protein ligase binding,ENSG00000161011
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0043130,ubiquitin binding,ENSG00000161011
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0016235,aggresome,ENSG00000161011
-Proteostasis,Pr,Proteostasis [Pr],#c8b269,GO:0016235,aggresome,ENSG00000000938
+biodomain,abbr,label,color,go_id,go_terms,n_symbol,symbol,ensembl_gene_id
```

Comment:
This file has no new test cases and is unchanged except for adding 2 new columns from the new biodomains file.